olclient/openlibrary.py

```diff
@@ -131,6 +131,19 @@ def save_many(self, docs, comment):
         doc_json = [doc.json() for doc in docs]
         return self.session.post('%s/api/save_many' % self.base_url, json.dumps(doc_json), headers=headers)
 
+    def delete_many(self, ol_ids: List[str], comment: str):
+        headers = {
+            'Opt': '"http://openlibrary.org/dev/docs/api"; ns=42',
+            '42-comment': comment
+        }
+        payload = [{
+            'key': f'/{self.get_type(ol_id)}s/{ol_id}',
+            'type': {
+                'key': '/type/delete'
+            }
+        } for ol_id in ol_ids]
+        return self.session.post(f'{self.base_url}/api/save_many', json.dumps(payload), headers=headers)
```

Comment:
Oh, might be nice to call self.save_many, to keep this DRY. Also we generally call these `olids` (no underscore)

```suggestion
    def delete_many(self, olids: List[str], comment: str):
        docs = [{
            'key': f'/{self.get_type(olid)}s/{olid}',
            'type': {
                'key': '/type/delete'
            }
        } for olid in olids]
        return self.save_many(docs, comment)
```

Reply:
Scratch that, it looks like save_many won't respond to a dict; it needs  document with a `.json()` method. So we should be able to do something like this (but not sure whether `Delete` accessible at this point, or needs some prefix).

```suggestion
    def delete_many(self, olids: List[str], comment: str):
        return self.save_many([Delete(olid) for olid in olids], comment)
```

Reply:
using `Delete` now, thanks for the suggestion